### PR TITLE
allow default FM rotation on init when `Keep FM rotation` is disabled

### PR DESCRIPTION
**Pull request by yparitcher**
fixes issue discovered in #6347

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/6348)

### CODE REVIEW

File: reader.lua [hunk 1]
```diff
@@ -277,7 +277,7 @@ if ARGV[argidx] and ARGV[argidx] ~= "" then
         local home_dir =
             G_reader_settings:readSetting("home_dir") or ARGV[argidx]
         UIManager:nextTick(function()
-            FileManager:showFiles(home_dir)
+            FileManager:showFiles(home_dir, "", true)
```
Comment (NiLuJe) [inline reader.lua:280]: `nil` instead of `""`, or does something wonky happen?

File: frontend/apps/filemanager/filemanager.lua [hunk 2]
```diff
@@ -70,10 +70,10 @@ function FileManager:onSetRotationMode(rotation)
     return true
 end
 
-function FileManager:setRotationMode()
+function FileManager:setRotationMode(init)
     local locked = G_reader_settings:readSetting("lock_rotation")
     local rotation_mode = G_reader_settings:readSetting("fm_rotation_mode") or 0
```
Comment (NiLuJe) [inline frontend/apps/filemanager/filemanager.lua:75]: Use the constant instead of 0, so I can get rid of that chunk in #6347 too ;p.

Comment (yparitcher) [reply frontend/apps/filemanager/filemanager.lua]: done

File: reader.lua (hunk 1 above)
Comment (yparitcher) [reply reader.lua]: > `nil` instead of `""`, or does something wonky happen?

Didn't check but assumed so.
Not relevant anyways now

**Pull request merged**